### PR TITLE
API Console

### PR DESCRIPTION
**Pull request by clackner-gpa**
This adds the Node Status (Nodes in OpenXDA and SystemCenter itself, single node only) and API to SC.
It's under SystemSettings> Nodes

### CODE REVIEW

File: Source/Applications/SystemCenter/Controllers/SystemCenter/AppHostController.cs [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+        {
+            string databaseName = "Not Available";
+            using (AdoDataConnection database = CreateDbConnection())
+            {
+                try
+                {
+                    databaseName = database.Connection.Database;
+                }
+                catch
+                { }
```
Comment (StephenCWills) [inline Source/Applications/SystemCenter/Controllers/SystemCenter/AppHostController.cs:230]: What kind of error are you trying to catch here?

File: Source/Applications/SystemCenter/wwwroot/Images/NodeTiles/SystemCenter.png
Comment (StephenCWills) [inline Source/Applications/SystemCenter/wwwroot/Images/NodeTiles/SystemCenter.png:1]: This image looks pretty bad on my system.

image

**Review by StephenCWills — APPROVED** [review]
@clackner-gpa Please review the changes I made to `AppHostController.DatabaseName`.

File: Source/Applications/SystemCenter/wwwroot/Images/NodeTiles/SystemCenter.png
Comment (clackner-gpa) [reply Source/Applications/SystemCenter/wwwroot/Images/NodeTiles/SystemCenter.png:1]: yea I need to get a higher resolution of the `SystemCenter` one but for now this should do

**Comment by clackner-gpa** [comment]
Changes look good. The `catch` was to ensure any errors in connecting to a valid database won't cause it to error out completely but that's covered in your change

File: Source/Applications/SystemCenter/wwwroot/Images/NodeTiles/SystemCenter.png
Comment (clackner-gpa) [reply Source/Applications/SystemCenter/wwwroot/Images/NodeTiles/SystemCenter.png:1]: Resolving for now we will get this picture resolved but I need the code changes ASAP

**Comment by StephenCWills** [comment]
Oops, I added a temp commit I was supposed to revert...

**Pull request merged**